Src/TestSample.BroadInt.Tests/Users/User.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Users
+
+Scenario: User is created
+	When a user is created with first name "first" and last name "last"
+	Then the a user is returned with the following properties
+    | FirstName | LastName |
+    | first     | last     |
+
+Scenario: Get a user by first and last name
+    Given a user exists with first name "first" and last name "last"
+	When a user is retrieved with first name "first" and last name "last"
+	Then the a user is returned with the following properties
+    | FirstName | LastName |
+    | first     | last     |
+
+Scenario: User is created with validation error

```

Comment:
I think we should keep negative scenarios out from our Broad Integration tests as we will be testing those scenarios well enough with component, micro/narrow integration tests and manual tests. Thoughts?
- Will the Broad Integration tests run after every build for a service (ex: Location) OR are they scheduled runs?
- What other tests run irrespective of whether or not changes were made to a service?